ui/app/appointments/controllers/manage/appointmentsCreateController.js

```diff
@@ -19,8 +19,22 @@ angular.module('bahmni.appointments')
             $scope.showStartTimes = [];
             $scope.showEndTimes = [];
 
+            var isProviderCurrentlyAvailableForAppointments = function (selectedProvider, config) {
+                var providers = config.providers;
+                return _.find(providers, function (provider) {
+                    return selectedProvider.uuid === provider.uuid;
+                });
+            };
             var init = function () {
                 wireAutocompleteEvents();
+                var providerCurrentAvailability = "False";
```

Comment:
Hi @maharjunm , on the line  `var providerCurrentAvailability = "False";` why is False written as a String,I mean in quotes?Any specific reason?

Reply:
Apart from the above comment,everything looks good so far.  
@mks-d Any comments?